crates/zksync/core/src/vm/inspect.rs

```diff
@@ -756,21 +756,25 @@ where
         .unwrap_or_default()
 }
 
-/// Maximum size allowed for factory_deps during create.
+/// Default maximum size allowed for factory_deps during create.
 /// We batch factory_deps till this upper limit if there are multiple deps.
 /// These batches are then deployed individually.
-const MAX_FACTORY_DEPENDENCIES_SIZE_BYTES: usize = 100_000;
+const DEFAULT_MAX_FACTORY_DEPENDENCIES_SIZE_BYTES: usize = 100_000;
 
 /// Batch factory deps on the basis of size.
 ///
 /// For large factory_deps the VM can run out of gas. To avoid this case we batch factory_deps
-/// on the basis of [MAX_FACTORY_DEPENDENCIES_SIZE_BYTES] and deploy all but the last batch
+/// on the basis of the max factory dependencies size and deploy all but the last batch
 /// via empty transactions, with the last one deployed normally via create.
 pub fn batch_factory_dependencies(mut factory_deps: Vec<Vec<u8>>) -> Vec<Vec<Vec<u8>>> {
+    let max_factory_deps_size_bytes = std::env::var("MAX_FACTORY_DEPENDENCIES_SIZE_BYTES")
```

Comment:
why not in the same place as before? doesn't change much but we don't need to do this check each time

Reply:
I figured it would be simpler and also it's probably a non issue performance wise. I can bring it back to the other one though as I don't really mind. Then if we make a PR to set it in config we can remove at that point.